### DISCUSSION TITLE
Where do i upload concepts now?

### DISCUSSION THREAD

**Discussion by Bonjonbon** [TheLastBen/fast-stable-diffusion#2287] [Q&A]
I don't know if it just me or what, but the concept cell in my colab is missing and I don't how to where to get it.

**Comment by TheLastBen** [comment]
no need for the concept cell, use the instance images cell

**Reply by Bonjonbon** [reply]
Oh okay, well I'm having a problem with mashed faces when creating a style so i thought that using concept art would help with that. But that might just be because i change it to150 in the Text Encoder Training Steps like in the guide instead of leaving it as is.
